src/app/voetbalpool/voetbalpool.lambda.ts

```diff
@@ -0,0 +1,51 @@
+import { DynamoDBClient, GetItemCommand, PutItemCommand } from '@aws-sdk/client-dynamodb';
+import { ApiGatewayV2Response, Response } from '@gemeentenijmegen/apigateway-http/lib/V2/Response';
+import { APIGatewayProxyEventV2 } from 'aws-lambda';
+
+const client = new DynamoDBClient({});
+
+export async function handler (event: APIGatewayProxyEventV2, _context: any) :Promise<ApiGatewayV2Response> {
+  try {
+
+    const body = event.body;
+    if (!body) {
+      return Response.error(400, 'No body');
+    }
+    if (body.length > 400) {
+      return Response.error(400, 'Too big');
+    }
+
+    const obj = JSON.parse(body);
+    if (!obj || !obj.email) {
+      return Response.error(400, 'Invalid body');
+    }
+
+    if (!obj.email.endsWith('@nijmegen.nl')) {
+      return Response.error(400, 'Not nijmegen');
+    }
+
+    const check = await client.send(new GetItemCommand({
```

Comment:
Is dit nodig, kun je bestaande niet gewoon overschrijven? Als dat kan hoeft de lambda ook alleen maar write-rechten te hebben misschien?

Reply:
Het idee is om een stand door te geven, dat mag je maar een keer doen. Als het overschreven kan worden moeten we op tijd een export maken… dit leek mij eenvoudiger. Er wordt verder alleen ok terug gegeven.